fvm/storage/primary/intersect.go

```diff
@@ -0,0 +1,42 @@
+package primary
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+)
+
+func intersectHelper[
+	T1 any,
+	T2 any,
+](
+	smallSet map[flow.RegisterID]T1,
+	largeSet map[flow.RegisterID]T2,
+) (
+	bool,
+	flow.RegisterID,
```

Comment:
Why is this returning a single register ID? the intersection could be more than one register.

Reply:
o, I wanted at least one register id for error messaging.  any one of them will do